### DISCUSSION TITLE
An error occurs when connecting to the StrongSwan server on Windows Server 2016.

### DISCUSSION THREAD

**Discussion by limitless** [strongswan/strongswan#1500] [Q&A]
Error message:
Error 809: The network connection between your computer and the VPN server could not be established because the remote server is not responding. This could be because one of the network devices (e.g, firewalls, NAT, routers, etc) between your computer and the remote server is not configured to allow VPN connections. Please contact your Administrator or your service provider to determine which device may be causing the problem.

Windows Server 2016 communicates through NAT.

Is there a solution to this issue?

**Comment by tobiasbrunner** [comment]
Obviously depends on what the problem is (could be a network/firewall issue, a config problem, or something else). Try to provide more information (see e.g. #196).
